src/components/title/_fields/identifiers/identifiers-fields.js

```diff
@@ -13,96 +12,101 @@ import {
 
 import { FormattedMessage } from 'react-intl';
 
-export default class IdentifiersFields extends Component {
-  validateId(value) {
-    let errors;
+import { IDENTIFIERS_FIELDS_VALUE_MAX_LENGTH } from '../../../../constants';
 
-    if (!value) {
-      errors = <FormattedMessage id="ui-eholdings.validate.errors.identifiers.noBlank" />;
-    }
+const validateId = (value) => {
+  let errors;
 
-    if (value && value.length >= 20) {
-      errors = <FormattedMessage id="ui-eholdings.validate.errors.identifiers.exceedsLength" />;
-    }
+  if (!value) {
+    errors = <FormattedMessage id="ui-eholdings.validate.errors.identifiers.noBlank" />;
+  }
 
-    return errors;
+  if (value?.length >= IDENTIFIERS_FIELDS_VALUE_MAX_LENGTH) {
+    errors = <FormattedMessage id="ui-eholdings.validate.errors.identifiers.exceedsLength" />;
   }
 
-  renderField = (identifier) => {
-    return (
-      <Row>
-        <Col
-          md
-          xs={12}
-          data-test-eholdings-identifiers-fields-type
-        >
-          <Field
-            name={`${identifier}.flattenedType`}
-            type="text"
-            component={Select}
-            autoFocus={Object.keys(identifier).length === 0}
-            label={<FormattedMessage id="ui-eholdings.type" />}
-          >
-            <FormattedMessage id="ui-eholdings.label.identifier.issnOnline">
-              {(message) => <option value="0">{message}</option>}
-            </FormattedMessage>
-            <FormattedMessage id="ui-eholdings.label.identifier.issnPrint">
-              {(message) => <option value="1">{message}</option>}
-            </FormattedMessage>
-            <FormattedMessage id="ui-eholdings.label.identifier.isbnOnline">
-              {(message) => <option value="2">{message}</option>}
-            </FormattedMessage>
-            <FormattedMessage id="ui-eholdings.label.identifier.isbnPrint">
-              {(message) => <option value="3">{message}</option>}
-            </FormattedMessage>
-          </Field>
-        </Col>
-        <Col
-          md
-          xs={12}
-          data-test-eholdings-identifiers-fields-id
+  return errors;
+};
+
+const renderField = (identifier) => {
```

Comment:
I think it's better to make this function as a part of `IdentifiersField` component